### PR TITLE
`changes.csv`: Clarify quoting

### PR DESCRIPTION
**Pull request by tschmidtb51**
- addresses parts of oasis-tcs/csaf#768
- change changes.csv to non-quoted
- clarify wording
- add normative reference to RFC 4180

### CODE REVIEW

File: csaf_2.1/prose/edit/src/distributing.md [hunk 1]
```diff
@@ -194,18 +194,25 @@ The index.txt file within MUST provide a list of all filenames of CSAF documents
 
 ### Requirement 13: changes.csv
 
-The file changes.csv MUST contain the filename as well as the value of `/document/tracking/current_release_date` for each
+The file `changes.csv` contains a list of CSAF documents in the current TLP level that were changed recently.
+Therefore, it MUST contain the filename as well as the value of `/document/tracking/current_release_date` for each
 CSAF document in the sub-directories without a heading; lines MUST be sorted by the `current_release_date` timestamp with the latest one first.
+The `changes.csv` SHALL be a valid comma separated values format as defined by [cite](#RFC4180) without double quotes.
+
+> Note: As a consequence of section [sec](#requirement-2-filename) Requirement 2 for filenames and section [sec](#requirement-11-one-folder-per-year)
```
Comment (tschmidtb51) [inline csaf_2.1/prose/edit/src/distributing.md:202]: @sthagen Should we refer to the titles of these sections or just leave it as it is right now?

If we want to refer to the titles, do you have a special label (e.g. `[title](#ref)`) to do that?

Comment (sthagen) [reply csaf_2.1/prose/edit/src/distributing.md:202]: I would leave it like that. No one reads only paper version of our work products and all digital user facing delivery items indicate the link target which nicely summarizes the title.

**Review by sthagen — APPROVED** [review]
LGTM.

**Comment by tschmidtb51** [comment]
Pending motion: https://groups.oasis-open.org/discussion/motion-for-830

**Comment by santosomar** [comment]
### Summary of Changes
1. **File: `csaf_2.1/prose/edit/src/distributing.md`**
   - Updated the requirements for `changes.csv` to clarify that it must contain filenames and release dates without quotes, sorted by timestamp.
   - Added notes explaining the requirement for the `changes.csv` format and its compatibility with CSAF 2.0.
   - Removed double quotes from example entries in `changes.csv`.

2. **File: `csaf_2.1/prose/edit/src/introduction-03-normative-references.md`**
   - Added a normative reference to RFC 4180, which defines the CSV format.

### Status
The motion has passed.  https://groups.oasis-open.org/discussion/motion-for-830

**Review by santosomar — APPROVED** [review]

**Pull request merged**